mlfinlab/labeling/labeling.py

```diff
@@ -67,11 +67,11 @@ def add_vertical_barrier(t_events, close, num_days=1):
 
     :param t_events: (series) series of events (symmetric CUSUM filter)
     :param close: (series) close prices
-    :param num_days: (int) maximum number of days a trade can be active
+    :param timedelta: (pd.Timedelta) maximum time a trade can be active
```

Comment:
What is the unit of time? days, seconds, bars?

Reply:
Will this break the notebooks which use a num_days = 1 parameter? I am thinking of the notebooks for chapter 3 on meta_labeling.

Reply:
Looking at the unit tests it looks like it will. `timedelta=pd.Timedelta('{}D'.format(days))`

There must be another clever way to deal with this. We need an implementation that doesn't break old code or breaks the code of people currently using this package in their production environments. 

Can we extend the funciton signiture? Should we create a new funciton with a different signiture?